packages/passport/sdk/src/zkEvm/personalSign.ts

```diff
@@ -38,7 +38,12 @@ export const personalSign = async ({
     throw new JsonRpcError(RpcErrorCode.INVALID_PARAMS, 'personal_sign requires the signer to be the from address');
   }
 
-  // Convert message into a string if it's a hex
+  if (typeof window !== 'undefined' && !window.Buffer) {
```

Comment:
👀 very interesting.. why does this work?

Reply:
The `hexToString` function called after this if statement uses `Buffer` which causes an issue with using the browser bundle as `Buffer` doesn't exist. Adding it to the window if Buffer isnt defined fixes the issue.